src/btwa_api/app/api/endpoints.py

```diff
@@ -2,18 +2,30 @@
 from fastapi.security import HTTPBasicCredentials
 
 from ..db.database import Database
+from ..db.firebase import Firebase, get_firebase
 from ..db.utils import get_db
 from ..security import check_handler_auth, security
 
 router = APIRouter()
 
 
-@router.get("/register/{guid}")
-def register(guid: str,
-             credentials: HTTPBasicCredentials = Depends(security),
-             db: Database = Depends(get_db)):
+@router.get("/proximity/{phone}")
```

Comment:
Is `phone` what is good to use as internal ID? Why not to use BUID or FUID? (is relevant also to `mark-infected` endpoint)

Reply:
This is the API for the frontend. The epidemiologist enter the phone number, so this is the input that we get. At least that is how I understood it :-)

Reply:
FUID is primary key here. You need to query Firestore first - get FUID based on phoneNumber

Reply:
Yes, that's how it currently works.